packages/e2e-tests/specs/experiments/template-part.test.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * WordPress dependencies
+ */
+import { visitAdminPage } from '@wordpress/e2e-test-utils';
+import { addQueryArgs } from '@wordpress/url';
+
+/**
+ * Internal dependencies
+ */
+import {
+	enableExperimentalFeatures,
+	disableExperimentalFeatures,
+} from '../../experimental-features';
+import { trashExistingPosts } from '../../config/setup-test-framework';
+
+describe( 'Template Part', () => {
+	beforeAll( async () => {
+		await enableExperimentalFeatures( [
+			'#gutenberg-full-site-editing',
+			'#gutenberg-full-site-editing-demo',
+		] );
+		await trashExistingPosts( 'wp_template' );
+		await trashExistingPosts( 'wp_template_part' );
+	} );
+	afterAll( async () => {
+		await trashExistingPosts( 'wp_template' );
+		await trashExistingPosts( 'wp_template_part' );
+		await disableExperimentalFeatures( [
+			'#gutenberg-full-site-editing',
+			'#gutenberg-full-site-editing-demo',
+		] );
+	} );
+	beforeEach( () =>
+		visitAdminPage(
+			'admin.php',
+			addQueryArgs( '', {
+				page: 'gutenberg-edit-site',
+			} ).slice( 1 )
+		)
```

Comment:
Was this copied from somewhere or expecting some future growth to the query arguments? 

Seems like it may be much simpler as:

```suggestion
		visitAdminPage( 'admin.php', 'page=gutenberg-edit-site' )
```

I also wonder if `visitAdminPage( 'admin.php?page=gutenberg-edit-site' )` can work as well. Sorta seems like it could, or should. The query argument passing is a bit awkward.

Reply:
It was copied. I thought it was a convention. I applied your suggestion.

>I also wonder if visitAdminPage( 'admin.php?page=gutenberg-edit-site' ) can work as well. Sorta seems like it could, or should. The query argument passing is a bit awkward.

It should, but that doesn't save much space and kind of bypasses the "correct" way of calling the function.